lib/libutil++/Makefile

```diff
@@ -0,0 +1,20 @@
+PACKAGE=	lib${LIB}
+LIB_CXX=	util++
+INTERNALLIB=	true
+SHLIB_MAJOR=	1
+SRCS=		stringf.cc
+
+MAN+=	freebsd::FILE_up.3 \
+	freebsd::addrinfo_up.3 \
+	freebsd::fd_up.3 \
+	freebsd::malloc_up.3 \
+	freebsd::nvlist_up.3 \
+	freebsd::pidfile.3 \
+	freebsd::stringf.3
```

Comment:
Colons in filenames are cool?

Reply:
As others have noted, some ports (such as Perl) already use colons in filenames.  I did have to fix bsd.man.mk (one of the commits in the branch)

Reply:
Hello! While I realize it might not be a priority here, I just wanted to leave a comment to let you know these recent changes to add files with : in the filename have broken git clone/pull on Windows (which doesn't allow these characters in the file names). I had the repo cloned on a Windows laptop so I could read the code, but now can no longer git pull the latest. It would be great if we could maintain that support. Thanks!

Reply:
@nibanks please open a ticket at bugs.freebsd.org .